lib/windows/winlean.nim

```diff
@@ -36,6 +36,8 @@ type
   DWORD* = int32
   PDWORD* = ptr DWORD
   LPINT* = ptr int32
+  ULONG_PTR* = uint
```

Comment:
I think we should keep things consistent and use signed integers everywhere as is done in this file. So you can just use ``PULONG`` instead of defining ``PULONG_PTR``.